lib/check_search.rb

```diff
@@ -241,7 +241,7 @@ def get_pg_results_for_media
       custom_conditions[k] = [@options[v]].flatten if @options.has_key?(v)
     end
     core_conditions.merge!({ archived: @options['archived'] })
-    core_conditions.merge!({ sources_count: 0 }) unless should_include_related_items?
+    core_conditions.merge!({ sources_count: 0 }) unless @options['show_similar']
```

Comment:
@melsawy why we need to keep this logic for PG-only queries? I saw you removed it for ES queries, as expected.

Reply:
For the PG (cases not hit ES) the `sources_count`  is the way we can know if this is a parent or child but ES we have a `parent_id` field 
so the cases for search by keyword, status, tags, etc we already hit ES so I think it's save to keep the condition for PG only